config/betterquesting/DefaultQuests.json

```diff
@@ -3519,7 +3751,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "All machines have 1 or more §cfront faces§r, and 1 §9output face§r.\nThe §cfront face§r should be obvious.\nThe §9output face§r is the face which has a dot or hole on it. By default, this is opposite the front face, or at the back of the machine as you place it.\n\nMachines can §aauto-output§r through their output face. To turn on auto-output, click the appropriate button in the GUI. Fluid and item auto-outputs are toggled separately. Further control over auto-outputs can be achieved with §aFilters§r and §aCovers§r, which are explained in the quest to the left of this one.\n\nRight-clicking a machine with a §aWrench§r changes the output side, and shift-right-clicking changes the front side. Keep in mind that the front side §ccannot§r also be the output side! \n\nBy default, items and fluids can be inserted through the output side, but shift-right-clicking with a §aScrewdriver§r can toggle this off. This could be useful for certain automation setups.\n\n\n§bLore:§r In base GTCE, the output side disallowed input by default. If you forgot to turn it on every time, have fun troubleshooting your setup",
+          "desc:8": "All machines typically have one §cFront Face§r, and one §9Output Face§r.\nThe §cFront Face§r should be obvious.\nThe §9Output Face§r is the face with a dot or hole on it. By default, this is at the back of the machine as you place it.\n\nMachines can §aauto-output§r through their output face. To turn on auto-output, click the appropriate button in the GUI. Fluid and item auto-outputs are toggled separately. Further control over auto-outputs can be achieved with §dFilters§r and §dCovers§r, which are explained in the quest to the left of this one.\n\nRight-clicking a machine with a §aWrench§r changes the output side, and shift-right-clicking changes the front side. Keep in mind that the front side §ccannot§r also be the output side! \n\nBy default, items and fluids cannot be inserted through the output side, but this can be toggled with a §9Screwdriver§r. Either right-click the cover directly, or shift-right-click on the grid to the side of the cover. This could be useful for certain automation setups.\nNote: At the time of writing this, the chat message for output side is actually the opposite of what it actually does, whoops!",
```

Comment:
```suggestion
          "desc:8": "All machines typically have one §cFront Face§r, and one §9Output Face§r.\nThe §cFront Face§r should be obvious.\nThe §9Output Face§r is the face with a dot or hole on it. By default, this is at the back of the machine as you place it.\n\nMachines can §aauto-output§r through their output face. To turn on auto-output, click the appropriate button in the GUI. Fluid and item auto-outputs are toggled separately. Further control over auto-outputs can be achieved with §dFilters§r and §dCovers§r, which are explained in the quest to the left of this one.\n\nRight-clicking a machine with a §aWrench§r changes the output side, and shift-right-clicking changes the front side. Keep in mind that the front side §ccannot§r also be the output side! \n\nBy default, items and fluids cannot be inserted through the output side, but this can be toggled with a §9Screwdriver§r. Either right-click the output icon directly, or shift-right-click on the grid corresponding to the side of the output. This could be useful for certain automation setups.\nNote: At the time of writing this, the chat message for output side is actually the opposite of what it actually does, whoops!",
```